spec/spec_helper.rb

```diff
@@ -272,6 +272,41 @@ def create_vm(**args)
       Vm.create(**args)
     end
 
+    def create_vm_etc(project, private_subnet, name)
```

Comment:
This method name is a bit vague, though I realize in this case it is hard to get a good descriptive name that isn't long. Maybe `create_hosted_vm`?

Reply:
seems easy to confuse with `create_vm_host` but makes sense